src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -2497,6 +2497,7 @@ def calculate_REE_costing_bounds(
                 doc="feedstock capacity of site",
                 units=pyunits.tonnes,
             )
+            b.capacity.fix(value(pyunits.convert(capacity, to_units=pyunits.tonnes)))
```

Comment:
You do not need the `value` or `convert` here - `b.capacity.fix(capacity)` will wokr as long as `capacity` is a Pyomo Var or Param or `pint` Quantity (which it appears to be).